hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/catalog/HoodieHiveCatalog.java

```diff
@@ -455,8 +456,9 @@ public void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
       throw new DatabaseNotExistException(getName(), tablePath.getDatabaseName());
     }
 
-    if (!table.getOptions().getOrDefault(CONNECTOR.key(), "").equalsIgnoreCase("hudi")) {
-      throw new HoodieCatalogException(String.format("The %s is not hoodie table", tablePath.getObjectName()));
```

Comment:
Should the `tablePath.getObjectName()` still be in the error message?

Reply:
This exeception is only throwed when using `create table options(connection = 'xxx')` and `xxx != HoodieTableFactory.FACTORY_ID`, user just need to know the correct FACTORY_ID, `tablePath.getObjectName()` no need to bo in the error message

Reply:
Got it.